exotations/solvers/exotica_ddp_solver/include/exotica_ddp_solver/abstract_ddp_solver.h

```diff
@@ -215,6 +220,16 @@ double AbstractDDPSolver<Initializer>::ForwardPass(const double alpha, Eigen::Ma
     return cost;
 }
 
+template <typename Initializer>
+Eigen::VectorXd AbstractDDPSolver<Initializer>::GetFeedbackControl(Eigen::VectorXd x, int t) const
```

Comment:
Eigen::VectorXdRefConst, const int, perhaps declare as inline?

Reply:
It's virtual, yes.